### DISCUSSION TITLE
How do I download all styles and models that are required for all the styles?

### DISCUSSION THREAD

**Discussion by gerroon** [lllyasviel/Fooocus#3541] [Q&A] [answered]
Hi

I want to use Fooocus on a totally offline computer and I want to make sure all the styles and the models (if necessary) to be downloaded?

thanks

**Comment by Yogesh-DevHub** [comment, marked as answer]
I check running without internet it works fine, do the installation on a computer with internet it will download the necessary files, generate a test image if possible. then just copy the main folder to a different computer without internet.

**Reply by mashb1t** [reply]
this or find a script for the most common use cases here: https://github.com/lllyasviel/Fooocus/issues/3310#issuecomment-2240058056
